### PR TITLE
new argument for the command line

### PR DESCRIPTION
**Pull request by RezaT1994**
I've added a new argument for the command line to write the documentation into a file.

```php
bin\phpcs --standard=Generic --generator=html --output-path=.\generic.html
```

### CODE REVIEW

File: src/Standards/Generic/Docs/Arrays/ArrayIndentStandard.xml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<documentation title="Array Indent">
```
Comment (jrfnl) [inline src/Standards/Generic/Docs/Arrays/ArrayIndentStandard.xml:1]: This file seems out of place in this PR ?

Comment (RezaT1994) [reply src/Standards/Generic/Docs/Arrays/ArrayIndentStandard.xml]: How can I add a new documentation?

Comment (jrfnl) [reply src/Standards/Generic/Docs/Arrays/ArrayIndentStandard.xml]: Separate PR ?

Comment (RezaT1994) [reply src/Standards/Generic/Docs/Arrays/ArrayIndentStandard.xml]: Yes

Comment (RezaT1994) [reply src/Standards/Generic/Docs/Arrays/ArrayIndentStandard.xml]: I just remove the this file in this PR.
So, should I add new documentation in a new PR?

**Comment by gsherwood** [comment]
I'm not sure this feature is required. It's not hard to get content into a file, and it's not something that is done often enough (like in CI) to make a dedicated CLI command worth it. Would be good to know what use case you are solving for though.

**Comment by RezaT1994** [comment]
I know it's not required. because we can just copy the content from terminal. But it's hard sometimes 🙁. The easy way is that we just provide a path to save the content to a file instead of copy and paste the content from terminal.

**Comment by jrfnl** [comment]
@RezaT1994 Have you tried piping the output to a file ? In my experience that saves you from having to manually copy/paste.

```
bin\phpcs --standard=Generic --generator=html > ./generic.html
```

The `>` basically says "take the output from this command and save it to a file with this name".

**Comment by RezaT1994** [comment]
Well I didn't know that. Thank you.
Looks like we didn't need that argument after all 😅

Should I close the PR?

**Comment by jrfnl** [comment]
> Looks like we didn't need that argument after all 😅

@RezaT1994 No worries, we all learn every day ;-)

> Should I close the PR?

You may as well. Would be nice if you would open a new PR for the docs for the ArrayIndent sniff which were originally included in this PR. 👍

**Comment by RezaT1994** [comment]
Ok I will

**Pull request closed without merging**